src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -32,7 +32,7 @@ public sealed class BinaryLogger : ILogger
         // version 7:
         //   - Include ProjectStartedEventArgs.GlobalProperties
         // version 8:
-        //   - new record kinds: EnvironmentVariableRead, PropertyReassignment, UninitializedPropertyRead
```

Comment:
@KirillOsenkov do you prefer a version number bump instead, to avoid a small window of time where version 8 would be broken?

Reply:
I can bump it. Will do next commit.